lib/shared/drive-constraints.js

```diff
@@ -404,9 +405,11 @@ exports.getDriveImageCompatibilityStatuses = (drive, image) => {
       message: exports.COMPATIBILITY_STATUS_MESSAGES.LOCKED
     })
   } else if (!_.isNil(drive) && !_.isNil(drive.size) && !exports.isDriveLargeEnough(drive, image)) {
+    const imageSize = _.get(image, [ 'size', 'final', 'estimation' ]) ? image.size.original : image.size.final.value
+    const relativeBytes = imageSize - drive.size
     statusList.push({
       type: exports.COMPATIBILITY_STATUS_TYPES.ERROR,
-      message: exports.COMPATIBILITY_STATUS_MESSAGES.TOO_SMALL
+      message: exports.COMPATIBILITY_STATUS_MESSAGES.TOO_SMALL.replace('XX', prettyBytes(relativeBytes))
```

Comment:
This is failing with `Prefer '_.replace' over the native function  lodash/prefer-lodash-method`

Reply:
Fixed